### DISCUSSION TITLE
v2&Vue2.0 获取具名插槽传递的值

### DISCUSSION THREAD

**Discussion by qq1037305420** [alibaba/formily#1534] [Q&A 2.x]
![image](https://user-images.githubusercontent.com/28857867/120295318-9eb15880-c2f9-11eb-8346-feff67c77b0a.png)

vue版本 x-content 具名插槽获取插槽传递过来的值
（看了看源码啥都没看懂。大概是太菜了 ;( ）

**Comment by MisicDemone** [comment]
目前暂不支持，由于x-content目前接受的是组件，所以插槽参数可能通过props传入。或者如果有更好的提案可以发出来讨论一下

**Reply by qq1037305420** [reply]
我觉得通过props传入没毛病。；）
